sasdata/data_util/manipulations.py

```diff
@@ -63,19 +63,18 @@ def get_q_compo(dx: float, dy: float, detector_distance: float, wavelength: floa
     return out
 
 
-def flip_phi(phi: float) -> float:
+def normalize_angle(phi: float, closed_at_end: bool = False) -> float:
     """
-    Force phi to be within the 0 <= to <= 2pi range by adding or subtracting
-    2pi as necessary
+    Force phi to be within the half-open interval 0 <= phi < 2pi, unless
+    closed_at_end is set to True, in which case the interval is 0 < phi <= 2pi.
```

Comment:
Why the difference and can you document those differences here?

Reply:
The `normalize_angle` method has been changed now, but I've supplied comments explaining the choices in the new version.